.github/workflows/rspec.yml

```diff
@@ -0,0 +1,21 @@
+name: RSpec
+
+on: [push, pull_request]
+
+jobs:
+  rspec:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: .ruby-version
```

Comment:
hmm... It needs to use several versions of ruby from 2.7.1 to current. Is it possible on Github Actions?